core/kazoo_json/include/kazoo_json.hrl

```diff
@@ -13,42 +13,35 @@
 
 -define(EMPTY_JSON_OBJECT, ?JSON_WRAPPER([])).
 
--type object() :: ?JSON_WRAPPER(json_proplist()) | ?EMPTY_JSON_OBJECT.
--type objects() :: [object()].
-
--type json_string() :: ne_binary() | atom().
--type json_strings() :: [json_string()].
+-type json_value() :: 'null'
```

Comment:
Could you revert the renaming? It hides actual changes and you didn't propagate it throughout the codebase.